### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by vectordiman**

### CODE REVIEW

**Comment by Exqzore** [comment]
Privet a gde lrb po aoks? asujdauuu

File: COSII/main.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    image_plot.show('Исходное изображение', 'Яркость исходного изображения', 256)
+
+    high_plot = plot.Plot(high_image, high_brightness)
+    high_plot.show('Отфильтрованное изображение', 'Яркость отфильтрованного изображения', 256)
+
+    # negative_plot = plot.Plot(negative_image, negative_brightness)
+    # negative_plot.show('Негативное изображение', 'Яркость негативного изображения', 256)
+    #
+    # sobel_plot = plot.Plot(sobel_image, sobel_brightness)
+    # sobel_plot.show('Отфильтрованное изображение', 'Яркость отфильтрованного изображения', 256)
```
Comment (Exqzore) [inline COSII/main.py:34]: zachem tut comments?

File: COSII/filter.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        size = result.shape
+        for i in range(1, size[0] - 1):
+            for j in range(1, size[1] - 1):
+                for k in range(0, size[2]):
+                    p = (self.points[i - 1][j - 1][k] + 2 * self.points[i][j - 1][k] + self.points[i + 1][j - 1][k]) - (
+                            self.points[i - 1][j + 1][k] + 2 * self.points[i][j + 1][k] + self.points[i + 1][j + 1][k])
+                    q = (self.points[i - 1][j - 1][k] + 2 * self.points[i - 1][j][k] + self.points[i - 1][j + 1][k]) - (
+                            self.points[i + 1][j - 1][k] + 2 * self.points[i + 1][j][k] + self.points[i + 1][j + 1][k])
+                    result[i][j][k] = min(255, numpy.sqrt(p ** 2 + q ** 2))
+        return Image.fromarray(result)
```
Comment (Exqzore) [inline COSII/filter.py:22]: chto tut proishodit?

File: COSII/plot.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+        ax1.imshow(self.image)
+        ax1.set(title=title1)
+        ax1.grid(False)
+
+        ax2.hist(self.brightness, bins, color='k')
+        ax2.set(title=title2)
+        ax2.grid(False)
+
+        matplotlib.pyplot.show()
```
Comment (vectordiman) [inline COSII/plot.py:25]: Vadim suck

**Pull request closed without merging**